lndr-backend/data/lndr-server.config.test

```diff
@@ -40,7 +40,7 @@ db {
 }
 
 # configuration used for calling CP's 'issueCredit' function
-execution-private-key = "7920ca01d3d1ac463dfd55b5ddfdcbb64ae31830f31be045ce2d51a305516a37"
+execution-private-key = "edc63d0e14b29aaa26c7585e962f93abb59bd7d8b01b585e073dc03d052a000b"
```

Comment:
Just to confirm, this private key is only used in test environments whem interacting with ganache?

Reply:
yes!